etc/profile-m-z/mirage.profile

```diff
@@ -11,6 +11,8 @@ noblacklist ${HOME}/.config/mirage
 noblacklist ${HOME}/.local/share/mirage
 noblacklist /sbin
 
+include allow-bin-sh.inc
```

Comment:
This can not work because `private-bin` does not contain `sh`. Was this tested?

Reply:
Yes, I'm going to add it back, I had some other small changes for matrix apps. It was a conflict.

Reply:
--build on the appimage says it tries to access ` x86_64-linux-gnu-ld.bfd,x86_64-linux-gnu-gcc-10` in private-bin amongs others.

Reply:
The previous profile seems to work fine for me but I'm adding the changes except that if in case it doesn't work.